tests/check_sssd_with_tlog.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Check with-tlog feature exists
+  command: authselect list-features sssd
+  register: __tlog_authselect_features
+  changed_when: false
+
+- name: Check if files domain is currently enabled
+  command: authselect current
+  register: __tlog_authselect_current
+  changed_when: false
+  failed_when: __tlog_authselect_current.rc not in [0, 2]
+
+- name: Read nsswitch.conf
+  slurp:
+    src: /etc/nsswitch.conf
+  register: __nsswitch_slurp
+
+- name: Decode nsswitch content
+  set_fact:
+    __nsswitch_contents: "{{ __nsswitch_slurp['content'] | b64decode }}"
+
+- name: Check if files domain enabled and nsswitch set correctly
```

Comment:
Should there be an opposite check for systems where `"with-tlog" not in __tlog_authselect_features.stdout`?

Reply:
If `with-tlog` feature is not available then we fallback to `with-files-domain` feature, which we already have a test for `tests/check_sssd_files_provider.yml`